docs/howto/openstack/nova/new-server.md

```diff
@@ -329,17 +371,17 @@ file first](../../getting-started/enable-openstack-cli.md).
     ```
 ## Connecting to the server console
 === "{{gui}}"
-    While viewing information regarding your server, you may get
-    its public IP  address (e.g., from the _Addresses_ tab) and connect to
-    it remotely. Alternatively, you may launch a web console and log in;
-    click on the three-dot icon on the right of the server header, and from
-    the pop-up menu that appears select _Remote Console_.
+    While viewing information regarding your server, you may get its public
+    IP  address (e.g., from the _Addresses_ tab) and connect to it remotely
+    via SSH. Alternatively, you may launch a web console and log in; click
+    on the three-dot icon on the right of the server header, and from the
+    pop-up menu that appears select _Remote Console_.
 
     ![Launch remote console](assets/new-server/shot-10.png)
 
-    A new window pops up, and that's your web console to your
-    {{brand}} server. Please note that this window cannot be
-    resized but can be opened on a new browser window or tab.
+    A new window pops up, and that's your web console to your {{brand}}
+    server. Please note that this window cannot be resized but can be opened
+    on a new browser window or tab.
```

Comment:
It just occurred to me that I typically open things _in_ a new browser window or tab, not _on_ one.

Reply:
Ja, ich auch :|